lib/utils/app_config.dart

```diff
@@ -49,6 +47,12 @@ FontWeight regular = FontWeight.w300;
 FontWeight bold = FontWeight.bold;
 FontWeight semiBold = FontWeight.w500;
 
+List<String> lang = [
```

Comment:
we should add more description on what this means, it is a configuration for 1.1.5 compatibility, a description would help here

Reply:
Added description for language list.